Cargo.toml

```diff
@@ -11,7 +11,7 @@ keywords = ["channel", "asynchronous", "mpmc", "golang", "select"]
 license = "Unlicense/MIT"
 
 [dependencies]
-rand = "0.3"
+rand = "0.4"
```

Comment:
You'll need to remove this. While I don't have the Travis CI configured right, this is going to bump the minimum Rust version required by `chan`, which will in turn require me to do a semver bump of `chan`. I'm not sure that I'm ready to do that.

Reply:
There will have been a very good reason for me doing this, as it's not something I would do likely, but unfortunately I've forgotten what it is. I'll see if I can dig up the reason.